bsp/boards/openmote-b/cx2538xf53.icf

```diff
@@ -0,0 +1,109 @@
+/*###ICF### Section handled by ICF editor, don't touch! ****/
```

Comment:
please explain where this files comes from, and justifty that isn't not an artifact.

Reply:
this is the linker file necessary for IAR compilation @changtengfei